tracer/src/Datadog.Trace/AppSec/SecuritySettings.cs

```diff
@@ -21,6 +21,8 @@ public SecuritySettings(IConfigurationSource source)
             var extraHeaders = source?.GetString(ConfigurationKeys.AppSecExtraHeaders);
             ExtraHeaders = !string.IsNullOrEmpty(extraHeaders) ? extraHeaders.Split(',') : Array.Empty<string>();
             KeepTraces = source?.GetBool(ConfigurationKeys.AppSecKeepTraces) ?? true;
+            var traceRateLimit = (source?.GetInt32(ConfigurationKeys.AppSecTraceRateLimit)).GetValueOrDefault(0);
+            TraceRateLimit = traceRateLimit > 0 ? traceRateLimit : 100;
```

Comment:
Seems weird that if you set this to 1 it works, but if you set it to 0, then it becomes 100. Did you mean `>= 0`?

Reply:
I see the RFC says _strictly positive_, but I think allowing zero makes sense. It means don't set sampling priority `UserKeep` on _any_ trace.

Reply:
> It means don't set sampling priority UserKeep on any trace.

To clarify, I meant "zero blocks everything" because even a single trace is over the limit. No special handling needed.